grant-project/zcash-grant-system
# Environment variable for explorer URL

Closes #74.

### What This Does

Adds an `EXPLORER_URL` variable to frontend and backend. This is optional and defaults to mainnet. The env examples have info for mainnet and testnet. There is no way to get an explorer for regtest (obviously) so this doesn't _really_ matter for dev.

### Steps to Test

* Run the branch without changing any env vars, and confirm it works without biting your head off.
* Create and send a contribution transaction.
    * Confirm the "view transaction" button on your profile takes you to the mainnet explorer.
    * Confirm the email sends you to the same mainnet explorer.
* Update your .env file with the testnet explorer URL and restart frontend. Confirm that the view transaction button on your profile now takes you to testnet explorer.
